From: KevinDuan <duankaiwen@kylinos.cn>
Date: Mon, 17 Aug 2026 06:39:50 +0000
Subject: Translated using Weblate (Japanese)

Currently translated at 94.8% (260 of 274 strings)

Translation: openkylin-huanghe/peony-qt
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-qt/ja/
---
 translations/peony-qt/peony-qt_ja.ts | 524 ++++++++++++++++++-----------------
 1 file changed, 265 insertions(+), 259 deletions(-)

diff --git a/translations/peony-qt/peony-qt_ja.ts b/translations/peony-qt/peony-qt_ja.ts
index c685f5b..6b5988c 100644
--- a/translations/peony-qt/peony-qt_ja.ts
+++ b/translations/peony-qt/peony-qt_ja.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../../src/windows/about-dialog.ui" line="32"/>
         <source>Dialog</source>
-        <translation type="unfinished"></translation>
+        <translation>ウィンドウ</translation>
     </message>
     <message>
         <location filename="../../src/windows/about-dialog.ui" line="88"/>
@@ -20,32 +20,32 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/windows/about-dialog.ui" line="115"/>
         <source>TextLabel</source>
-        <translation type="unfinished"></translation>
+        <translation>タブ</translation>
     </message>
     <message>
         <location filename="../../src/windows/about-dialog.cpp" line="47"/>
         <location filename="../../src/windows/about-dialog.cpp" line="171"/>
         <source>Service &amp; Support: </source>
-        <translation type="unfinished"></translation>
+        <translation>サービスおよびサポートチーム: </translation>
     </message>
     <message>
         <location filename="../../src/windows/about-dialog.cpp" line="46"/>
         <location filename="../../src/windows/about-dialog.cpp" line="101"/>
         <location filename="../../src/windows/about-dialog.cpp" line="117"/>
         <source>Peony</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャー</translation>
     </message>
     <message>
         <location filename="../../src/windows/about-dialog.cpp" line="51"/>
         <location filename="../../src/windows/about-dialog.cpp" line="132"/>
         <source>Peony is a graphical software to help users manage system files. It provides common file operation functions for users, such as file viewing, file copy, paste, cut, delete, rename, file selection, application opening, file search, file sorting, file preview, etc. it is convenient for users to manage system files intuitively on the interface.</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャーは、ユーザーがシステムファイルを管理するためのグラフィカルソフトウェアであり、ファイルの表示、コピー、貼り付け、切り取り、削除、名前の変更、ファイルの開き方選択、検索、並べ替え、プレビューなど、一般的なファイル操作機能を提供し、ユーザーが直感的にシステムファイルを管理できるようにします。</translation>
     </message>
     <message>
         <location filename="../../src/windows/about-dialog.cpp" line="50"/>
         <location filename="../../src/windows/about-dialog.cpp" line="120"/>
         <source>Version number: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>バージョン番号: %1</translation>
     </message>
 </context>
 <context>
@@ -53,22 +53,22 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/file-label-box.cpp" line="81"/>
         <source>Open In New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいウィンドウで開く</translation>
     </message>
     <message>
         <location filename="../../src/control/file-label-box.cpp" line="71"/>
         <source>Open In New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいタブで開く</translation>
     </message>
     <message>
         <location filename="../../src/control/file-label-box.cpp" line="93"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>名前の変更</translation>
     </message>
     <message>
         <location filename="../../src/control/file-label-box.cpp" line="98"/>
         <source>Edit Color</source>
-        <translation type="unfinished"></translation>
+        <translation>色を編集</translation>
     </message>
 </context>
 <context>
@@ -79,18 +79,18 @@ p, li { white-space: pre-wrap; }
         <location filename="../../src/control/file-label-menu.cpp" line="343"/>
         <location filename="../../src/control/file-label-menu.cpp" line="370"/>
         <source>Add labels to &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>「%1」にラベルを追加</translation>
     </message>
     <message>
         <location filename="../../src/control/file-label-menu.cpp" line="279"/>
         <location filename="../../src/control/file-label-menu.cpp" line="377"/>
         <source>Add labels to %1 items</source>
-        <translation type="unfinished"></translation>
+        <translation>%1個のアイテムにラベルを追加</translation>
     </message>
     <message>
         <location filename="../../src/control/file-label-menu.cpp" line="313"/>
         <source>Label Management</source>
-        <translation type="unfinished"></translation>
+        <translation>ラベル管理</translation>
     </message>
 </context>
 <context>
@@ -98,100 +98,100 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/header-bar.cpp" line="142"/>
         <source>Go Back</source>
-        <translation type="unfinished"></translation>
+        <translation>戻る</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="150"/>
         <source>Go Forward</source>
-        <translation type="unfinished"></translation>
+        <translation>進む</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="388"/>
         <source>Option</source>
-        <translation type="unfinished"></translation>
+        <translation>オプション</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="517"/>
         <source>Operate Tips</source>
-        <translation type="unfinished"></translation>
+        <translation>操作ヒント</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="169"/>
         <source>Go Up</source>
-        <translation type="unfinished"></translation>
+        <translation>上の階層</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="414"/>
         <source>&amp;Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>コピー</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="417"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>コピー</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="429"/>
         <source>&amp;Cut</source>
-        <translation type="unfinished"></translation>
+        <translation>切り取り</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="432"/>
         <source>Cut</source>
-        <translation type="unfinished"></translation>
+        <translation>切り取り</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="438"/>
         <source>&amp;Select All</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="442"/>
         <location filename="../../src/control/header-bar.cpp" line="454"/>
         <source>Select All</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="459"/>
         <location filename="../../src/control/header-bar.cpp" line="1011"/>
         <location filename="../../src/control/header-bar.cpp" line="1029"/>
         <source>Deselect All</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択解除</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="464"/>
         <source>&amp;Delete to trash</source>
-        <translation type="unfinished"></translation>
+        <translation>ごみ箱に削除</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="467"/>
         <source>Delete to trash</source>
-        <translation type="unfinished"></translation>
+        <translation>ごみ箱に削除</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="518"/>
         <source>Don&apos;t find any terminal, please install at least one terminal!</source>
-        <translation type="unfinished"></translation>
+        <translation>どの端末プラグインも見つかりませんでした。少なくとも1つのプラグインがインストールされていることを確認してください。</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="813"/>
         <location filename="../../src/control/header-bar.cpp" line="1014"/>
         <location filename="../../src/control/header-bar.cpp" line="1024"/>
         <source>Select All Item</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="823"/>
         <location filename="../../src/control/header-bar.cpp" line="958"/>
         <location filename="../../src/control/header-bar.cpp" line="997"/>
         <source>Select</source>
-        <translation type="unfinished"></translation>
+        <translation>選択</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="794"/>
         <location filename="../../src/control/header-bar.cpp" line="795"/>
         <source>Restore</source>
-        <translation type="unfinished"></translation>
+        <translation>元に戻す</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="799"/>
@@ -202,27 +202,27 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/header-bar.cpp" line="1000"/>
         <source>Select Done</source>
-        <translation type="unfinished"></translation>
+        <translation>完了</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="850"/>
         <source>MoveTo</source>
-        <translation type="unfinished"></translation>
+        <translation>移動先</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="859"/>
         <source>CopyTo</source>
-        <translation type="unfinished"></translation>
+        <translation>コピー先</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="868"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>削除</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="935"/>
         <source>Select path</source>
-        <translation type="unfinished"></translation>
+        <translation>パスを選択</translation>
     </message>
 </context>
 <context>
@@ -235,7 +235,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/header-bar.cpp" line="1448"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>閉じる</translation>
     </message>
 </context>
 <context>
@@ -243,12 +243,12 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-navigation-side-bar.cpp" line="180"/>
         <source>Open In &amp;New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいウィンドウで開く(&amp;N)</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-navigation-side-bar.cpp" line="204"/>
         <source>Open In New &amp;Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいタブで開く(&amp;T)</translation>
     </message>
 </context>
 <context>
@@ -256,7 +256,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-navigation-side-bar.cpp" line="516"/>
         <source>Files</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル</translation>
     </message>
 </context>
 <context>
@@ -264,17 +264,17 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/windows/main-window.cpp" line="1085"/>
         <source>File Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャー</translation>
     </message>
     <message>
         <location filename="../../src/windows/main-window.cpp" line="539"/>
         <source>Undo</source>
-        <translation type="unfinished"></translation>
+        <translation>元に戻す</translation>
     </message>
     <message>
         <location filename="../../src/windows/main-window.cpp" line="546"/>
         <source>Redo</source>
-        <translation type="unfinished"></translation>
+        <translation>やり直し</translation>
     </message>
     <message>
         <location filename="../../src/windows/main-window.cpp" line="980"/>
@@ -284,17 +284,17 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/windows/main-window.cpp" line="980"/>
         <source>This operation is not supported.</source>
-        <translation type="unfinished"></translation>
+        <translation>この操作はサポートされていません。</translation>
     </message>
     <message>
         <location filename="../../src/windows/main-window.cpp" line="1083"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>検索</translation>
     </message>
     <message>
         <location filename="../../src/windows/main-window.cpp" line="1098"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいフォルダーを作成</translation>
     </message>
 </context>
 <context>
@@ -307,45 +307,45 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="726"/>
         <source>This operation is not supported.</source>
-        <translation type="unfinished"></translation>
+        <translation>この操作はサポートされていません。</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="209"/>
         <location filename="../../src/control/navigation-side-bar.cpp" line="765"/>
         <location filename="../../src/control/navigation-side-bar.cpp" line="783"/>
         <source>Tips</source>
-        <translation type="unfinished"></translation>
+        <translation>ヒント</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="209"/>
         <source>The device is in busy state, please perform this operation later.</source>
-        <translation type="unfinished"></translation>
+        <translation>デバイスがビジーです。後でこの操作を実行してください。</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="765"/>
         <source>This is an abnormal Udisk, please fix it or format it</source>
-        <translation type="unfinished"></translation>
+        <translation>これは異常なUSBメモリです。修復またはフォーマットしてください</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="783"/>
         <source>This is an empty drive, please insert a Disc.</source>
-        <translation type="unfinished"></translation>
+        <translation>これは空の光ドライブです。ディスクを挿入してください。</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="303"/>
         <source>Open In New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいウィンドウで開く</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="284"/>
         <location filename="../../src/control/navigation-side-bar.cpp" line="326"/>
         <source>Can not open %1, %2</source>
-        <translation type="unfinished"></translation>
+        <translation>%1, %2 を開けません</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="257"/>
         <source>Open In New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいタブで開く</translation>
     </message>
 </context>
 <context>
@@ -353,12 +353,12 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="1089"/>
         <source>Manager tags...</source>
-        <translation type="unfinished"></translation>
+        <translation>識別モード</translation>
     </message>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="1092"/>
         <source>More tags...</source>
-        <translation type="unfinished"></translation>
+        <translation>その他のラベル</translation>
     </message>
 </context>
 <context>
@@ -366,77 +366,77 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="84"/>
         <source>Keep Allow</source>
-        <translation type="unfinished"></translation>
+        <translation>ウィンドウを最前面に固定</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="89"/>
         <source>Show Hidden</source>
-        <translation type="unfinished"></translation>
+        <translation>隠しファイルを表示</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="107"/>
         <source>Show File Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル拡張子を表示</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="112"/>
         <source>Show Create Time</source>
-        <translation type="unfinished"></translation>
+        <translation>作成時間を表示</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="119"/>
         <source>Show Relative Time</source>
-        <translation type="unfinished"></translation>
+        <translation>相対時間を表示</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="135"/>
         <source>Forbid thumbnailing</source>
-        <translation type="unfinished"></translation>
+        <translation>サムネイルを無効にする</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="160"/>
         <source>Resident in Backend</source>
-        <translation type="unfinished"></translation>
+        <translation>バックグラウンドで常駐</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="169"/>
         <source>Parallel Operations</source>
-        <translation type="unfinished"></translation>
+        <translation>操作の並列実行を許可</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="176"/>
         <source>Set samba password</source>
-        <translation type="unfinished"></translation>
+        <translation>Samba共有パスワードを設定</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="217"/>
         <source>Tips</source>
-        <translation type="unfinished"></translation>
+        <translation>ヒント</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="218"/>
         <source>The user already has a samba password, do you need to reset the samba password?</source>
-        <translation type="unfinished"></translation>
+        <translation>ユーザーはすでにSamba共有パスワードを設定しています。再設定しますか？</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="220"/>
         <source>Ok</source>
-        <translation type="unfinished"></translation>
+        <translation>OK</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="221"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>キャンセル</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="233"/>
         <source>Samba set user password</source>
-        <translation type="unfinished"></translation>
+        <translation>Samba共有パスワードを設定</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="231"/>
         <source>Samba password:</source>
-        <translation type="unfinished"></translation>
+        <translation>共有パスワード:</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="241"/>
@@ -447,47 +447,47 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="241"/>
         <source>Samba set password failed, Please re-enter!</source>
-        <translation type="unfinished"></translation>
+        <translation>共有パスワードの設定に失敗しました。再入力してください</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="252"/>
         <source>Shared configuration service exception, please confirm if there is an ongoing shared configuration operation, or please reset the share!</source>
-        <translation type="unfinished"></translation>
+        <translation>共有設定サービスに異常があります。既に進行中の共有設定操作があるか確認するか、再度共有を設定してください！</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="263"/>
         <source>Open each folder in a new window</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいウィンドウでフォルダーを開く</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="269"/>
         <source>Plugin manager Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>プラグイン管理設定</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="276"/>
         <source>Show Network</source>
-        <translation type="unfinished"></translation>
+        <translation>ネットワークプレースを表示</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="285"/>
         <source>Connect to Server</source>
-        <translation type="unfinished"></translation>
+        <translation>リモートサーバーに接続</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="314"/>
         <source>Use current view for all folders</source>
-        <translation type="unfinished"></translation>
+        <translation>すべてのフォルダーに現在のビューを使用</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="334"/>
         <source>Help</source>
-        <translation type="unfinished"></translation>
+        <translation>ヘルプ</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="338"/>
         <source>About</source>
-        <translation type="unfinished"></translation>
+        <translation>について</translation>
     </message>
 </context>
 <context>
@@ -495,27 +495,27 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="436"/>
         <source>Edit</source>
-        <translation type="unfinished"></translation>
+        <translation>編集</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="447"/>
         <source>copy</source>
-        <translation type="unfinished"></translation>
+        <translation>コピー</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="456"/>
         <source>paste</source>
-        <translation type="unfinished"></translation>
+        <translation>貼り付け</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="465"/>
         <source>cut</source>
-        <translation type="unfinished"></translation>
+        <translation>切り取り</translation>
     </message>
     <message>
         <location filename="../../src/control/operation-menu.cpp" line="474"/>
         <source>trash</source>
-        <translation type="unfinished"></translation>
+        <translation>削除</translation>
     </message>
 </context>
 <context>
@@ -523,22 +523,22 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/tab-widget.h" line="372"/>
         <source>type</source>
-        <translation type="unfinished"></translation>
+        <translation>種類</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="372"/>
         <source>file size</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルサイズ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="372"/>
         <source>modify time</source>
-        <translation type="unfinished"></translation>
+        <translation>変更時間</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="372"/>
         <source>name</source>
-        <translation type="unfinished"></translation>
+        <translation>名前</translation>
     </message>
 </context>
 <context>
@@ -546,12 +546,12 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-cloud-item.cpp" line="40"/>
         <source>CloudStorage</source>
-        <translation type="unfinished"></translation>
+        <translation>クラウドストレージ</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-cloud-item.cpp" line="55"/>
         <source>CloudFile</source>
-        <translation type="unfinished"></translation>
+        <translation>クラウドファイル</translation>
     </message>
 </context>
 <context>
@@ -559,7 +559,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-favorite-item.cpp" line="46"/>
         <source>Favorites</source>
-        <translation type="unfinished"></translation>
+        <translation>クイックアクセス</translation>
     </message>
 </context>
 <context>
@@ -567,17 +567,17 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="60"/>
         <source>Computer</source>
-        <translation type="unfinished"></translation>
+        <translation>コンピュータ</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="79"/>
         <source>文件系统</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルシステム</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="80"/>
         <source>System Disk</source>
-        <translation type="unfinished"></translation>
+        <translation>システムディスク</translation>
     </message>
 </context>
 <context>
@@ -590,28 +590,28 @@ p, li { white-space: pre-wrap; }
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="174"/>
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="242"/>
         <source>Properties</source>
-        <translation type="unfinished"></translation>
+        <translation>プロパティ</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="82"/>
         <source>Delete Symbolic</source>
-        <translation type="unfinished"></translation>
+        <translation>リンクを削除</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="132"/>
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="234"/>
         <source>Unmount</source>
-        <translation type="unfinished"></translation>
+        <translation>アンインストール</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="151"/>
         <source>Eject</source>
-        <translation type="unfinished"></translation>
+        <translation>ポップアップ</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-menu.cpp" line="203"/>
         <source>format</source>
-        <translation type="unfinished"></translation>
+        <translation>フォーマット</translation>
     </message>
 </context>
 <context>
@@ -619,7 +619,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-personal-item.cpp" line="45"/>
         <source>Personal</source>
-        <translation type="unfinished"></translation>
+        <translation>個人</translation>
     </message>
 </context>
 <context>
@@ -627,7 +627,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-separator-item.h" line="70"/>
         <source>(No Sub Directory)</source>
-        <translation type="unfinished"></translation>
+        <translation>（サブディレクトリなし）</translation>
     </message>
 </context>
 <context>
@@ -635,7 +635,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="722"/>
         <source>User Disk</source>
-        <translation type="unfinished"></translation>
+        <translation>ユーザーディスク</translation>
     </message>
 </context>
 <context>
@@ -643,7 +643,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/tablet-side-bar-factory.cpp" line="60"/>
         <source>Intel Side Bar</source>
-        <translation type="unfinished"></translation>
+        <translation>Intelサイドバー</translation>
     </message>
 </context>
 <context>
@@ -651,17 +651,17 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/windows/trash-warn-dialog.cpp" line="101"/>
         <source>Are you sure that you want to delete these files? Once you start a deletion, the files deleting will never be restored again.</source>
-        <translation type="unfinished"></translation>
+        <translation>これらのファイルを削除してもよろしいですか？削除を開始すると、これらのファイルは復元できなくなります。</translation>
     </message>
     <message>
         <location filename="../../src/windows/trash-warn-dialog.cpp" line="114"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>削除</translation>
     </message>
     <message>
         <location filename="../../src/windows/trash-warn-dialog.cpp" line="115"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>キャンセル</translation>
     </message>
 </context>
 <context>
@@ -669,17 +669,17 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/peony-application.cpp" line="177"/>
         <source>peony-qt</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャー</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.cpp" line="184"/>
         <source>Files or directories to open</source>
-        <translation type="unfinished"></translation>
+        <translation>開く必要のあるファイルまたはフォルダー</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.cpp" line="184"/>
         <source>[FILE1, FILE2,...]</source>
-        <translation type="unfinished"></translation>
+        <translation>[ファイル1、ファイル2...]</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.cpp" line="231"/>
@@ -693,12 +693,16 @@ p, li { white-space: pre-wrap; }
 1. Peony-Qt might be running as root, that means you have the higher permission and can do some things which normally forbidden. But, you should learn that if you were in a root, the virtual file system will lose some featrue such as you can not use &quot;My Computer&quot;, the theme and icons might also went wrong. So, run peony-qt in a root is not recommended.
 
 2. You are using a non-qt theme for your system but you didn&apos;t install the platform theme plugin for qt&apos;s applications. If you are using gtk-theme, try installing the qt5-gtk2-platformtheme package to resolve this problem.</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャがシステムアイコンテーマを取得できません。考えられる原因は以下の通りです：
+
+1. 管理者ユーザーとしてファイルマネージャを実行しているためです。これは高い権限を得ることを意味しますが、同時に通常ユーザーでのみ利用可能な機能（「マイコンピュータ」やシステムテーマなど）を失うことになります。特別な事情がない限り、管理者ユーザーでファイルマネージャを開かないでください。
+
+2. 使用中のシステムテーマがQtのデフォルトでサポートされているテーマではなく、関連するプラットフォームプラグインがインストールされていない可能性があります。Gtkテーマをシステムテーマとして使用している場合、この問題を解決するためにqt5-gtk2-platformthemeのインストールを試みてください。</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.cpp" line="531"/>
         <source>Peony Qt</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャー</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.cpp" line="532"/>
@@ -708,22 +712,22 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/peony-application.h" line="70"/>
         <source>Close all peony-qt windows and quit</source>
-        <translation type="unfinished"></translation>
+        <translation>すべてのウィンドウを閉じて終了</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.h" line="71"/>
         <source>Show items</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルのディレクトリを開いてそれらを選択</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.h" line="72"/>
         <source>Show folders</source>
-        <translation type="unfinished"></translation>
+        <translation>フォルダ内のコンテンツを表示</translation>
     </message>
     <message>
         <location filename="../../src/peony-application.h" line="73"/>
         <source>Show properties</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルプロパティウィンドウを開く</translation>
     </message>
 </context>
 <context>
@@ -731,74 +735,76 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="451"/>
         <source>Force unmount failed</source>
-        <translation type="unfinished"></translation>
+        <translation>強制アンマウントに失敗しました</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="451"/>
         <source>Error: %1
 </source>
-        <translation type="unfinished"></translation>
+        <translation>エラー：%1
+</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="455"/>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="494"/>
         <source>Data synchronization is complete,the device has been unmount successfully!</source>
-        <translation type="unfinished"></translation>
+        <translation>データの同期が完了し、デバイスは安全に取り外されました！</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="473"/>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="478"/>
         <source>Unmount failed</source>
-        <translation type="unfinished"></translation>
+        <translation>アンマウント失敗</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="473"/>
         <source>Unable to unmount it, you may need to close some programs, such as: GParted etc.</source>
-        <translation type="unfinished"></translation>
+        <translation>デバイスをアンマウントできません。GPartedなどのアプリケーションを閉じてから再度試してください。</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="478"/>
         <source>Error: %1
 Do you want to unmount forcely?</source>
-        <translation type="unfinished"></translation>
+        <translation>エラー：%1 
+強制アンマウントしてもよろしいですか？</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="630"/>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="684"/>
         <source>Eject failed</source>
-        <translation type="unfinished"></translation>
+        <translation>ポップアップ失敗</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="631"/>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="685"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>キャンセル</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="632"/>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="686"/>
         <source>Eject Anyway</source>
-        <translation type="unfinished"></translation>
+        <translation>強制的にポップアップ</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="644"/>
         <source>Data synchronization is complete and the device can be safely unplugged!</source>
-        <translation type="unfinished"></translation>
+        <translation>データの同期が完了しました。デバイスを安全に取り外すことができます。</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="682"/>
         <source>Unable to eject %1</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 を取り出せません</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="700"/>
         <source>PeonyNotify</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャーメッセージ</translation>
     </message>
     <message>
         <location filename="../../src/control/intel/intel-side-bar-file-system-item.cpp" line="701"/>
         <source>File Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャー</translation>
     </message>
     <message>
         <source>Are you sure you want to permanently delete this file?</source>
@@ -818,12 +824,12 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="34"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="38"/>
         <source>Time Modified</source>
-        <translation type="unfinished"></translation>
+        <translation>変更日時</translation>
     </message>
     <message>
         <source>Time Created</source>
@@ -832,94 +838,94 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="46"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルサイズ</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="50"/>
         <source>Original Path</source>
-        <translation type="unfinished"></translation>
+        <translation>元のパス</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="55"/>
         <source>Path</source>
-        <translation type="unfinished"></translation>
+        <translation>パス</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="61"/>
         <source>Intelligent sort</source>
-        <translation type="unfinished"></translation>
+        <translation>スマートソート</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="66"/>
         <source>Add Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>追加方法順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="69"/>
         <source>Manual Add Priority</source>
-        <translation type="unfinished"></translation>
+        <translation>手動追加優先</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="74"/>
         <source>Conditional Screening Priority</source>
-        <translation type="unfinished"></translation>
+        <translation>条件フィルター優先</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="79"/>
         <source>Manually added only</source>
-        <translation type="unfinished"></translation>
+        <translation>手動でのみ追加</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="84"/>
         <source>Conditional additions only</source>
-        <translation type="unfinished"></translation>
+        <translation>条件フィルターのみ</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="100"/>
         <location filename="../../src/control/sort-type-menu.cpp" line="167"/>
         <source>Descending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>降順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="105"/>
         <location filename="../../src/control/sort-type-menu.cpp" line="168"/>
         <source>Ascending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>昇順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="116"/>
         <source>Use current sorting for all folders</source>
-        <translation type="unfinished"></translation>
+        <translation>すべてのディレクトリで現在のソート順を使用</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="152"/>
         <source>By %1</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 で並べ替え</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="161"/>
         <source>Newest to oldest</source>
-        <translation type="unfinished"></translation>
+        <translation>新しい順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="162"/>
         <source>Oldest to newest</source>
-        <translation type="unfinished"></translation>
+        <translation>古い順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="164"/>
         <source>Files from large to small</source>
-        <translation type="unfinished"></translation>
+        <translation>大きい順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="165"/>
         <source>Files from small to large</source>
-        <translation type="unfinished"></translation>
+        <translation>小さい順</translation>
     </message>
     <message>
         <location filename="../../src/control/sort-type-menu.cpp" line="42"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルタイプ</translation>
     </message>
 </context>
 <context>
@@ -927,23 +933,23 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-status-bar.cpp" line="126"/>
         <source> %1 items </source>
-        <translation type="unfinished"></translation>
+        <translation> %1 個の項目 </translation>
     </message>
     <message>
         <location filename="../../src/control/tab-status-bar.cpp" line="113"/>
         <location filename="../../src/control/tab-status-bar.cpp" line="225"/>
         <source>Searching for files ...</source>
-        <translation type="unfinished"></translation>
+        <translation>検索中...</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-status-bar.cpp" line="162"/>
         <source> selected %1 items    %2</source>
-        <translation type="unfinished"></translation>
+        <translation> %1 個の項目を選択    %2</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-status-bar.cpp" line="164"/>
         <source> selected %1 items</source>
-        <translation type="unfinished"></translation>
+        <translation> %1 個の項目を選択</translation>
     </message>
 </context>
 <context>
@@ -951,50 +957,50 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="259"/>
         <source>Show all tabs</source>
-        <translation type="unfinished"></translation>
+        <translation>すべてのタブを表示</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="239"/>
         <source>New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいタブ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="310"/>
         <source>Trash</source>
-        <translation type="unfinished"></translation>
+        <translation>ごみ箱</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="314"/>
         <source>Clear</source>
-        <translation type="unfinished"></translation>
+        <translation>クリア</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="319"/>
         <source>Recover</source>
-        <translation type="unfinished"></translation>
+        <translation>復元</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="642"/>
         <location filename="../../src/control/tab-widget.cpp" line="664"/>
         <location filename="../../src/control/tab-widget.cpp" line="976"/>
         <source>Search Condition</source>
-        <translation type="unfinished"></translation>
+        <translation>検索条件</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="643"/>
         <location filename="../../src/control/tab-widget.cpp" line="661"/>
         <source>Clear Conditions</source>
-        <translation type="unfinished"></translation>
+        <translation>条件をクリア</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="783"/>
         <source>Need to save changes to current space conditions?</source>
-        <translation type="unfinished"></translation>
+        <translation>現在のスペース条件への変更を保存しますか？</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="784"/>
         <source>Not Save</source>
-        <translation type="unfinished"></translation>
+        <translation>保存しない</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="785"/>
@@ -1005,77 +1011,77 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="920"/>
         <source>Location:</source>
-        <translation type="unfinished"></translation>
+        <translation>範囲：</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="983"/>
         <source>Save as Space</source>
-        <translation type="unfinished"></translation>
+        <translation>スペースとして保存</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="1005"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>キャンセル</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="1013"/>
         <source>Edit Conditiones</source>
-        <translation type="unfinished"></translation>
+        <translation>編集スペース</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2771"/>
         <source>Contains</source>
-        <translation type="unfinished"></translation>
+        <translation>含む</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2771"/>
         <source>Not Contains</source>
-        <translation type="unfinished"></translation>
+        <translation>含まない</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2772"/>
         <source>FileName and FileContent</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名とファイル内容</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2772"/>
         <source>Only FileName</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2825"/>
         <source>File smart tags</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルスマートタグ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2903"/>
         <source>Search Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>検索設定</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2903"/>
         <source>After the creation of the index, the next search can get the results of the document content containing the search term, during which you can exit the page at any time, we will continue to complete the creation in the background.</source>
-        <translation type="unfinished"></translation>
+        <translation>インデックスを作成すると、次回の検索でファイル内容に検索ワードが含まれる結果を得ることができます。作成中でもページをいつでも離れることができます。バックグラウンドで処理を続けます。</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="971"/>
         <source>Computer</source>
-        <translation type="unfinished"></translation>
+        <translation>コンピューター</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="380"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルの種類</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="380"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルサイズ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="380"/>
         <source>File Label</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル識別子</translation>
     </message>
     <message>
         <source>Filter</source>
@@ -1084,7 +1090,7 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="1205"/>
         <source>Select Path</source>
-        <translation type="unfinished"></translation>
+        <translation>パスを選択</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="1896"/>
@@ -1094,22 +1100,22 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="1899"/>
         <source>Error</source>
-        <translation type="unfinished"></translation>
+        <translation>エラー</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2348"/>
         <source>Opening such files is not currently supported</source>
-        <translation type="unfinished"></translation>
+        <translation>この種類のファイルを開くことは一時的にサポートされていません</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2363"/>
         <source>Open failed</source>
-        <translation type="unfinished"></translation>
+        <translation>開けませんでした</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="2364"/>
         <source>Open directory failed, you have no permission!</source>
-        <translation type="unfinished"></translation>
+        <translation>フォルダーを開けませんでした。このディレクトリへの権限がありません！</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="373"/>
@@ -1117,68 +1123,68 @@ Do you want to unmount forcely?</source>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>all</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="373"/>
         <source>file folder</source>
-        <translation type="unfinished"></translation>
+        <translation>フォルダ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="373"/>
         <location filename="../../src/control/tab-widget.h" line="375"/>
         <source>image</source>
-        <translation type="unfinished"></translation>
+        <translation>画像</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="373"/>
         <location filename="../../src/control/tab-widget.h" line="375"/>
         <source>video</source>
-        <translation type="unfinished"></translation>
+        <translation>ビデオ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="374"/>
         <location filename="../../src/control/tab-widget.h" line="376"/>
         <source>text file</source>
-        <translation type="unfinished"></translation>
+        <translation>テキスト文書</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="374"/>
         <location filename="../../src/control/tab-widget.h" line="376"/>
         <source>audio</source>
-        <translation type="unfinished"></translation>
+        <translation>オーディオ</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="374"/>
         <location filename="../../src/control/tab-widget.h" line="376"/>
         <source>others</source>
-        <translation type="unfinished"></translation>
+        <translation>その他</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.cpp" line="1033"/>
         <source>Satisfy All Conditions</source>
-        <translation type="unfinished"></translation>
+        <translation>すべての条件を満たす</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="374"/>
         <location filename="../../src/control/tab-widget.h" line="376"/>
         <source>wps file</source>
-        <translation type="unfinished"></translation>
+        <translation>WPSファイル</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>today</source>
-        <translation type="unfinished"></translation>
+        <translation>今日</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>this week</source>
-        <translation type="unfinished"></translation>
+        <translation>今週</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>this month</source>
-        <translation type="unfinished"></translation>
+        <translation>今月</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
@@ -1188,22 +1194,22 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>yesterday</source>
-        <translation type="unfinished"></translation>
+        <translation>昨日</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>last week</source>
-        <translation type="unfinished"></translation>
+        <translation>先週</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>last month</source>
-        <translation type="unfinished"></translation>
+        <translation>先月</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
         <source>last year</source>
-        <translation type="unfinished"></translation>
+        <translation>昨年</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="377"/>
@@ -1213,32 +1219,32 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.h" line="379"/>
         <source>file name and content</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名と内容</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="379"/>
         <source>file name</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="380"/>
         <source>Modify Time</source>
-        <translation type="unfinished"></translation>
+        <translation>変更時間</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="380"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="380"/>
         <source>Condition Match Rule</source>
-        <translation type="unfinished"></translation>
+        <translation>一致ルール</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>tiny(0-16K)</source>
-        <translation type="unfinished"></translation>
+        <translation>極小(0-16K)</translation>
     </message>
     <message>
         <source>type</source>
@@ -1259,7 +1265,7 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>small(16k-1M)</source>
-        <translation type="unfinished"></translation>
+        <translation>非常に小さい(16k-1M)</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
@@ -1269,22 +1275,22 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>medium(1M-128M)</source>
-        <translation type="unfinished"></translation>
+        <translation>中程度(1M-128M)</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>big(128M-1G)</source>
-        <translation type="unfinished"></translation>
+        <translation>大（128MB～1GB）</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>large(1-4G)</source>
-        <translation type="unfinished"></translation>
+        <translation>巨大（1～4GB）</translation>
     </message>
     <message>
         <location filename="../../src/control/tab-widget.h" line="378"/>
         <source>great(&gt;4G)</source>
-        <translation type="unfinished"></translation>
+        <translation>巨大(&gt;4G)</translation>
     </message>
 </context>
 <context>
@@ -1292,7 +1298,7 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/navigation-side-bar.cpp" line="1234"/>
         <source>Peony</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルマネージャー</translation>
     </message>
 </context>
 <context>
@@ -1300,32 +1306,32 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="413"/>
         <source>New</source>
-        <translation type="unfinished"></translation>
+        <translation>新規作成</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="426"/>
         <source>Cut</source>
-        <translation type="unfinished"></translation>
+        <translation>カット</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="439"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>コピー</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="482"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>貼り付け</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="511"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>名前の変更</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="528"/>
         <source>Compress</source>
-        <translation type="unfinished"></translation>
+        <translation>圧縮</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="571"/>
@@ -1333,143 +1339,143 @@ Do you want to unmount forcely?</source>
         <location filename="../../src/control/toolbar-manager.cpp" line="1731"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="1763"/>
         <source>Delete</source>
-        <translation type="unfinished"></translation>
+        <translation>削除</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="783"/>
         <source>Ai Question Answer</source>
-        <translation type="unfinished"></translation>
+        <translation>AI質問</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="841"/>
         <source>AI has completed the analysis %1/%2 files</source>
-        <translation type="unfinished"></translation>
+        <translation>AIが%1/%2ファイルを解析しました</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="896"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="2666"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="2683"/>
         <source>Select All</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="897"/>
         <source>Select all</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="909"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="2762"/>
         <source>MultiSelect</source>
-        <translation type="unfinished"></translation>
+        <translation>複数選択</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="1480"/>
         <source>Intelligent Space</source>
-        <translation type="unfinished"></translation>
+        <translation>スマートスペース</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2060"/>
         <source>Connect a server</source>
-        <translation type="unfinished"></translation>
+        <translation>リモートサーバーに接続</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2147"/>
         <source>Open In New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいタブで開く</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2162"/>
         <source>Open In New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいウィンドウで開く</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2185"/>
         <source>Unmount</source>
-        <translation type="unfinished"></translation>
+        <translation>アンマウント</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2246"/>
         <source>Eject</source>
-        <translation type="unfinished"></translation>
+        <translation>ポップアップ</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2271"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="2313"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="2339"/>
         <source>Format</source>
-        <translation type="unfinished"></translation>
+        <translation>フォーマット</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2542"/>
         <source>Sort By</source>
-        <translation type="unfinished"></translation>
+        <translation>並べ替えタイプ</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2548"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイル名</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2549"/>
         <source>Time Modified</source>
-        <translation type="unfinished"></translation>
+        <translation>変更時間</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2550"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルタイプ</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2551"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>ファイルサイズ</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2552"/>
         <source>Original Path</source>
-        <translation type="unfinished"></translation>
+        <translation>元のパス</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2573"/>
         <source>Descending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>降順</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2573"/>
         <source>Ascending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>昇順</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2576"/>
         <source>Newest to oldest</source>
-        <translation type="unfinished"></translation>
+        <translation>新しい順</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2576"/>
         <source>Oldest to newest</source>
-        <translation type="unfinished"></translation>
+        <translation>古い順</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2579"/>
         <source>Files from large to small</source>
-        <translation type="unfinished"></translation>
+        <translation>大きい順</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2579"/>
         <source>Files from small to large</source>
-        <translation type="unfinished"></translation>
+        <translation>小さい順</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2664"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="2689"/>
         <source>Deselect All</source>
-        <translation type="unfinished"></translation>
+        <translation>すべて選択解除</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2757"/>
         <source>Exit MultiSelect</source>
-        <translation type="unfinished"></translation>
+        <translation>複数選択を終了</translation>
     </message>
     <message>
         <source>burndata</source>
@@ -1478,12 +1484,12 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2378"/>
         <source>Property</source>
-        <translation type="unfinished"></translation>
+        <translation>プロパティ</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="2380"/>
         <source>You have to mount this volume first</source>
-        <translation type="unfinished"></translation>
+        <translation>まずパーティションをマウントする必要があります</translation>
     </message>
     <message>
         <source>Cancel</source>
@@ -1492,37 +1498,37 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="594"/>
         <source>Icon View</source>
-        <translation type="unfinished"></translation>
+        <translation>アイコンビュー</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="611"/>
         <source>List View</source>
-        <translation type="unfinished"></translation>
+        <translation>リストビュー</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="730"/>
         <source>Add File Label</source>
-        <translation type="unfinished"></translation>
+        <translation>カラーラベルを追加</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="770"/>
         <source>Exit Edit Conditions</source>
-        <translation type="unfinished"></translation>
+        <translation>スペースの編集を終了</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="757"/>
         <source>Edit Conditions</source>
-        <translation type="unfinished"></translation>
+        <translation>スペースを編集</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="868"/>
         <source>Exit Search</source>
-        <translation type="unfinished"></translation>
+        <translation>検索を終了</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="923"/>
         <source>More</source>
-        <translation type="unfinished"></translation>
+        <translation>もっと見る</translation>
     </message>
     <message>
         <source>Empty File</source>
@@ -1531,22 +1537,22 @@ Do you want to unmount forcely?</source>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="1310"/>
         <source>Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>フォルダー</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="1314"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>新しいフォルダを作成</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="1326"/>
         <source>Text Document</source>
-        <translation type="unfinished"></translation>
+        <translation>テキストドキュメント</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="1754"/>
         <source>Delete to trash</source>
-        <translation type="unfinished"></translation>
+        <translation>ごみ箱に削除</translation>
     </message>
     <message>
         <location filename="../../src/control/toolbar-manager.cpp" line="1765"/>
@@ -1554,7 +1560,7 @@ Do you want to unmount forcely?</source>
         <location filename="../../src/control/toolbar-manager.cpp" line="1778"/>
         <location filename="../../src/control/toolbar-manager.cpp" line="1783"/>
         <source>Delete forever</source>
-        <translation type="unfinished"></translation>
+        <translation>完全に削除</translation>
     </message>
 </context>
 <context>
@@ -1563,7 +1569,7 @@ Do you want to unmount forcely?</source>
         <location filename="../../src/control/header-bar.cpp" line="1223"/>
         <location filename="../../src/control/header-bar.cpp" line="1330"/>
         <source>Option</source>
-        <translation type="unfinished"></translation>
+        <translation>オプション</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="1236"/>
@@ -1575,7 +1581,7 @@ Do you want to unmount forcely?</source>
         <location filename="../../src/control/header-bar.cpp" line="1260"/>
         <location filename="../../src/control/header-bar.cpp" line="1336"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>閉じる</translation>
     </message>
     <message>
         <location filename="../../src/control/header-bar.cpp" line="1334"/>
